code_examples/core_features/did/07_full_did_batch.ts

```diff
@@ -23,30 +21,39 @@ function getRandomCType(): Kilt.CType {
 }
 
 export async function batchCTypeCreationExtrinsics(
-  keystore: Kilt.Did.DemoKeystore,
   api: ApiPromise,
-  submitterAccount: KeyringPair,
-  fullDid: Kilt.Did.FullDidDetails,
-  resolveOn: Kilt.SubscriptionPromise.ResultEvaluator = Kilt.BlockchainUtils
+  submitterAccount: Kilt.KiltKeyringPair,
+  fullDid: Kilt.DidDetails,
+  signCallback: Kilt.SignCallback,
+  resolveOn: Kilt.SubscriptionPromise.ResultEvaluator = Kilt.Blockchain
     .IS_FINALIZED
 ): Promise<void> {
   // Create two random demo CTypes
   const ctype1 = getRandomCType()
-  const ctype1CreationTx = await ctype1.getStoreTx()
+  const ctype1CreationTx = await Kilt.CType.getStoreTx(ctype1)
   const ctype2 = getRandomCType()
-  const ctype2CreationTx = await ctype2.getStoreTx()
+  const ctype2CreationTx = await Kilt.CType.getStoreTx(ctype2)
 
   // Create the DID-signed batch
-  const batch = await new Kilt.Did.DidBatchBuilder(api, fullDid)
-    .addMultipleExtrinsics([ctype1CreationTx, ctype2CreationTx])
-    .build(keystore, submitterAccount.address)
+  const authorisedBatch = await Kilt.Did.authorizeBatch({
```

Comment:
Could do that for consistency
```suggestion
  const authorizedBatch = await Kilt.Did.authorizeBatch({
```

Reply:
https://github.com/KILTprotocol/docs/pull/140/commits/a1abe307a00c339b899ae91b52694529a22bf4fc